topics/quickstart.md

```diff
@@ -28,7 +28,7 @@ $ valkey-cli -h 127.0.0.1 -p 6379
 
 ## Store and retrieve data
 
-Valkey stands for Remote Dictionary Server. You can use the same data types as in your local programming environment but on the server side within Valkey.
+Valkey is a remote dictionary server. You can use the same data types as in your local programming environment but on the server side within Valkey.
```

Comment:
We could even say "remote data structure server" or something to avoid the old acronym, but I'll just merge this simple fix for now. Feel free to improve the texts more! They could use some work.